src/NetAnalyzers/Core/Microsoft.NetCore.Analyzers/xlf/MicrosoftNetCoreAnalyzersResources.cs.xlf

```diff
@@ -417,6 +417,21 @@
         <target state="translated">Toto přetížení metody Add je potenciálně nebezpečné, protože může překládat nebezpečné externí odkazy.</target>
         <note />
       </trans-unit>
+      <trans-unit id="DoNotAlwaysSkipTokenValidationInDelegates">
```

Comment:
Not familiar with the localization process here. Didn't see any reference to this in documentation, let me know if there are other steps I need to do regarding this. Thanks.

Reply:
Localizers will localize later.